Tasks/JenkinsDownloadArtifacts/task.json

```diff
@@ -39,11 +39,14 @@
         },
         {
             "name": "jobName",
-            "type": "string",
+            "type": "pickList",
```

Comment:
Isn't updating string to pickList a breaking change? What if the user has something that can't be retrieved by the data bindings?

Reply:
It works fine since the pickList is editable, user can provide any other value as input also.